src/utils/drizzle.ts

```diff
@@ -0,0 +1,18 @@
+import { sql } from "drizzle-orm";
+import { customType } from "drizzle-orm/sqlite-core";
+
+export const stringSet = customType<{
+	data: Set<string>;
+	driverData: string;
+	default: true;
+}>({
+	dataType() {
+		return "stringSet";
```

Comment:
This should probably just be `text`. SQLite will interpret this as an unknown type and give it a type affinity of `numeric`, which we do not want.

Reply:
oh I see, I was confused what this was for 😄